.github/workflows/build-and-release.yml

```
@@ -10,13 +10,22 @@ jobs:
     - uses: actions/checkout@v3
     - name: Set up Python
       uses: actions/setup-python@v4
+      with:
+        python-version: "3.11"
     - name: Install python dependencies
       run: |
         python -m pip install --upgrade pip build
     - name: Build sdist
       run: |
         python -m build -s
-    - name: Publish package
+    - name: Publish distribution 📦 to Test PyPI
+      uses: pypa/gh-action-pypi-publish@release/v1
+      if: startsWith(github.ref, 'refs/tags') != true
```

```suggestion
      if: "!startsWith(github.ref, 'refs/tags')"
```
[Source](https://github.com/orgs/community/discussions/25017#discussioncomment-3246242) - maybe it's a bit more idiomatic?

---

I think `!= true` is fine and more verbose since the `!` can be easily missed.

---

As you like.